core/domain/question_services.py

```diff
@@ -217,3 +227,242 @@ def update_question(
     updated_question = apply_change_list(question_id, change_list)
     _save_question(
         committer_id, updated_question, change_list, commit_message)
+
+
+def get_new_question_id():
+    """Returns a new question id.
+
+    Returns:
+        str. A new question id.
+    """
+    return question_models.QuestionModel.get_new_id('')
+
+
+def create_question_summary(
+        question_id, creator_id, status):
+    """Creates and stores a summary of the given question.
+
+    Args:
+        question_id: str. ID of the question.
+        creator_id: str. The user ID of the creator of the question.
+        status: str. The status of the question.
+    """
+    question = get_question_by_id(question_id)
+    question_summary = compute_summary_of_question(
+        question, creator_id, status)
+    save_question_summary(question_summary)
+
+
+def compute_summary_of_question(question, creator_id, status):
+    """Create a QuestionSummary domain object for a given Question domain
+    object and return it.
+
+    Args:
+        question: Question. The question object for which the summary
+            is to be computed.
+        creator_id: str. The user ID of the creator of the question.
+        status: str. The status of the question.
+
+    Returns:
+        QuestionSummary. The computed summary for the given question.
+    """
+    question_html_data = question.question_data['content']['html']
+    question_summary = question_domain.QuestionSummary(
+        question.question_id, creator_id, question.language_code,
+        status, question_html_data
+    )
+
+    return question_summary
+
+
+def save_question_summary(question_summary):
+    """Save a question summary domain object as a QuestionSummaryModel
+    entity in the datastore.
+
+    Args:
+        question_summary: The question summary object to be saved in the
+            datastore.
+    """
+    question_summary_model = question_models.QuestionSummaryModel(
+        id=question_summary.id,
+        creator_id=question_summary.creator_id,
+        language_code=question_summary.language_code,
+        status=question_summary.status,
+        question_model_last_updated=question_summary.last_updated,
+        question_model_created_on=question_summary.created_on,
+        question_html_data=question_summary.question_html_data
+    )
+
+    question_summary_model.put()
+
+
+def send_question_for_review(committer_id, question_id):
```

Comment:
Sorry for not catching this earlier, but why not have this in apply_change_list itself, as separate commands? That would look a lot cleaner.

Reply:
In apply_change_list we are updating QuestionModel. Here we are updating QuestionSummaryModel